images/bay-elasticsearch/Dockerfile

```diff
@@ -1,30 +1,3 @@
-FROM uselagoon/elasticsearch-6:latest
-
-WORKDIR /tmp
-
-RUN yum install unzip -y \
-    && yum install java-1.8.0-openjdk-devel -y \
-    && wget https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/archive/v1.16.21_es6.2.4.zip \
-    && unzip v1.16.21_es6.2.4.zip \
-    && mv elasticsearch-readonlyrest-plugin-1.16.21_es6.2.4 readonlyrest
-
-COPY build.gradle /tmp/readonlyrest/core/build.gradle
-COPY gradle.properties /tmp/readonlyrest/es61x/gradle.properties
-COPY settings.gradle /tmp/readonlyrest/settings.gradle
-
-WORKDIR /tmp/readonlyrest
-
-RUN ./gradlew  es61x:ror '-PesVersion=6.2.4'
-
-WORKDIR /usr/share/elasticsearch
-
-COPY readonlyrest/readonlyrest.yml config/readonlyrest.yml
-
-RUN bin/elasticsearch-plugin install file:///tmp/readonlyrest/es61x/build/distributions/readonlyrest-1.16.21_es6.2.4.zip --batch \
-    && chown elasticsearch:root config/readonlyrest.yml \
-    && chown -R elasticsearch:root plugins/readonlyrest \
-    && yum erase java-1.8.0-openjdk-devel -y \
-    && yum erase unzip -y\
-    && rm -f /tmp/v1.16.21_es6.2.4.zip \
-    && rm -rf /tmp/readonlyrest
+FROM elasticsearch:8.5.3
 
+RUN /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch ingest-attachment
```

Comment:
Could you install these oplugins too
-    "analysis-kuromoji"
-    "analysis-icu"

Reply:
@nicksantamaria Thanks for checking; I have updated the docker file to pick the advice plugins.